### PR TITLE
chore(docs): doc-ui readme update

### PR DESCRIPTION
**Pull request by ynnenu**
4. Documentation
- [x] Storybook documentation has been created/updated

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,24 +0,0 @@
-.DS_Store
```
Comment (platon-rov) [inline .gitignore:1]: Probable deleted by mistake.

**Review by InnaAtanasova — APPROVED** [review]

File: packages/doc-ui/README.md [hunk 2]
```diff
@@ -1,7 +1,35 @@
...
+<Versions>: displays all the older versions of the page in a list
+<Footer>: a block of text that appears at the foot of each page
+
+This library was generated with [Nx](https://nx.dev) in order to speed up the existing overflow by running only the affected tasks affected by the current PR. 
+
+## Getting Started 
+
+### 1. doc-ui as part of the Fundamental Library Styles
+doc-ui package is part of the Fundamental Library Styles. If you have installed the Fundamental Library Styles in your project, you automatically get the doc-ui package.
+
```
Comment (g-cheishvili) [inline packages/doc-ui/README.md:22]: no

File: packages/doc-ui/README.md [hunk 3]
```diff
@@ -1,7 +1,35 @@
...
+### 2. doc-ui on its own
+
+doc-ui package can stand on its own. To intall the doc-ui package run:
+
+```bash
+ npm i @fundamental-styles/doc-ui
+ ```
+
+## Running Unit Tests
```
Comment (g-cheishvili) [inline packages/doc-ui/README.md:33]: It is not published

File: packages/doc-ui/README.md [hunk 4]
```diff
@@ -1,7 +1,35 @@
...
+<Subtitle>: the secondary name of the composition
+<TOC>: the table of content which shows all the titles and subtitles along with the number of the page destination
+<InfoLabels>: displays the information of the label
+<Description>: gives longer and more detailed information about the composition
+<VisibleStories>: gives an overview of the stories shown on the page
+<Versions>: displays all the older versions of the page in a list
+<Footer>: a block of text that appears at the foot of each page
+
+This library was generated with [Nx](https://nx.dev) in order to speed up the existing overflow by running only the affected tasks affected by the current PR. 
+
```
Comment (g-cheishvili) [inline packages/doc-ui/README.md:17]: Excessive info

**Review by g-cheishvili — CHANGES_REQUESTED** [review]

**Review by platon-rov — COMMENTED** [review]
Seems like we have unified PR https://github.com/SAP/fundamental-styles/pull/4018 for all the readmes, should we then keep this one?

**Pull request closed without merging**